From 15ead2a109c14b86cef5a0790c49728f872b4ec6 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Sun, 17 Apr 2016 12:44:16 +0300
Subject: [PATCH 0180/1345] gic: a37xx: added GIC virtual maintenance
 interrupt

commit  63c5f7a5960595db07493c06a57b7806f05def4f from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This interrupt is used by the KVM hypervisor to inject interrupts
to guest OS

Change-Id: I059cfdc1bab6daa47085a19183a98b9d1d46c95e
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29106
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index a170957..e9c5f94 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -383,7 +383,7 @@
 				interrupt-controller;
 				reg = <0x1d00000 0x10000>, /* GICD */
 				      <0x1d40000 0x40000>; /* GICR */
-				interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
+				interrupts = <GIC_PPI 9 (GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>;
 			};
 		};
 
-- 
1.7.9.5

